mafia.conf

```diff
@@ -0,0 +1,53 @@
+server {
```

Comment:
i'm not familiar with uwsgi and docker, but this .conf looks like just a bunch of server paths with restrictions?